SECURITY.md

```diff
@@ -0,0 +1,60 @@
+# Security Policy
+## Supported Versions
+Current supported versions for Gravitee Access Management:
+| Version | Supported          |
+| ------- | ------------------ |
+| LTS:    |                    |
```

Comment:
Wouldn't it be more interesting to target to the online doc for the supporting versions rather than having to maintain the version directly in the file  🤔 ?
I believe we are maintaining the same way all the products? (Like APIM for instance has the same LTS as AM?)

Having to maintain versions would be prone to forget to modify the file.
Requesting @tcompiegne

Reply:
Yes, def and I would agree 100% with that @remisultan, but until then can we please put this one in?
@tiffanymarchi can support us on this too and she def should be in the loop.

Reply:
Awesome.
I have created an issue so we don't forget about it: https://github.com/gravitee-io/issues/issues/6156

In the meantime, I'll merge this.